app/prednasky-workshopy/page.js

```
@@ -85,7 +85,10 @@ export default async function PrednaskyWorhsopy() {
                         className={`inline`}
                         size={18}
                       />
-                      {w.speaker.name}
+
+                      <span
+                        dangerouslySetInnerHTML={{ __html: w.speaker.name }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)